From 9fa6e82a0b0f6b6ed259c1e1efbe4a1b33c19569 Mon Sep 17 00:00:00 2001
From: Sabri Ünal <yakushabb@gmail.com>
Date: Thu, 28 May 2026 16:50:49 +0300
Subject: [PATCH] Fix build erro

```
CMake Error at CMakeLists.txt:2 (cmake_policy):
  Policy CMP0043 may not be set to OLD behavior because this version of CMake
  no longer supports it.  The policy was introduced in CMake version 3.0.0,
  and use of NEW behavior is now required.

  Please either update your CMakeLists.txt files to conform to the new
  behavior or use an older version of CMake that still supports the old
  behavior.  Run cmake --help-policy CMP0043 for more information.
```

Signed-off-by: Sabri Ünal <yakushabb@gmail.com>
---
 CMakeLists.txt | 1 -
 1 file changed, 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 2669b65..202860b 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,5 +1,4 @@
 CMAKE_MINIMUM_REQUIRED(VERSION 3.1.0 FATAL_ERROR)
-cmake_policy(SET CMP0043 OLD) # cmake --help-policy CMP0043
 
 PROJECT(NitrokeyApp LANGUAGES CXX)
 SET(PROJECT_VERSION "1.4.2")
--
libgit2 1.7.2

